roles/openshift_master/tasks/main.yml

```diff
@@ -180,6 +180,28 @@
 - name: Install the systemd units
   include: systemd_units.yml
 
+- name: Checking for journald.conf
+  stat: path=/etc/systemd/journald.conf
+  register: journald_conf_file
+
+- name: Update journald setup
+  replace:
```

Comment:
Nice timing, @soltysh  I was just looking into this problem in my env.

Would it have been more clean to drop a templated file into `/etc/systemd/journald.conf.d` instead?

Reply:
@dlbewley not sure, I'd defer to @sdodson as the main ansible installer maintainer.

Reply:
@dlbewley Yeah, that's probably a nice improvement. PR for that ? :-) Given this just merged I wouldn't worry about attempting to migrate anyone from how this PR modified things to the dropin file

Reply:
Other drop ins we've named 'openshift-ansible.conf' FWIW

Reply:
i noticed that in `dnsmasq.d`, @sdodson i'll try to find some time to do that PR. don't hold your breath :)